packages/fxa-settings/src/components/Settings/FlowContainer/index.stories.tsx

```diff
@@ -27,3 +28,91 @@ export const Default = () => (
     {MOCK_CONTENT}
   </FlowContainer>
 );
+
+export const WizardExample = () => {
+  const [currentStep, setCurrentStep] = useState(1);
+  /*
+    In this wizard pattern, you'll want to have equality checks which control which step is being displayed (like,
+      { currentStep === 1 && (<YourViewHere/> )}
+    ).
+    Each step will be built with the <FlowContainer /> component as the outer wrapper.  Each step will at minimum expect the props `currentStep` and `setCurrentStep`, received from this component. The `currentStep` value and `setCurrentStep` method will be passed in to the `<FlowContainer/>` as props of the same name. Each step will want to have the <ProgressBar /> component as a child.
+    We have filler components hardcoded in here (the Wizard pattern seems too simple to justify making it into a reusable component), but we can replace these as we create the real views.
```

Comment:
We might not need this comment? Looks like it might be leftover from the previous iteration of the wizard

Reply:
I left it because I thought it might be useful, but now that I'm looking at it it is a little silly to have since this is the FlowContainer storybook lol. I'll remove the comment!